### PR TITLE
fix: MD003/heading-style/header-style

### PR DESCRIPTION
**Pull request by nschonni**
Use ATX instead of ATX_CLOSED to match config

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6921743](https://github.com/nschonni/PowerShell-Docs/pull/3828/commits/6921743c3557af830b82fbcec666391a58f372ff)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6921743](https://github.com/nschonni/PowerShell-Docs/pull/3828/commits/6921743c3557af830b82fbcec666391a58f372ff)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 50 sec(s)_

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md [hunk 1]
```diff
@@ -96,106 +96,6 @@ Beginning in Windows PowerShell 3.0, the Enable-PSRemoting cmdlet can enable
 remoting on client and server versions of Windows on private, domain, and
 public networks.
 
-On server versions of Windows with private and domain networks, the
```
Comment (nschonni) [inline reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: It appears that part of the file got pasted in here

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6921743](https://github.com/nschonni/PowerShell-Docs/pull/3828/commits/6921743c3557af830b82fbcec666391a58f372ff)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_powershell_exe?view=powershell-3.0&branch=pr-en-us-3828) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_windows_powershell_5.1?view=powershell-5.1&branch=pr-en-us-3828) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_windows_powershell_5.1.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_remote_requirements?view=powershell-6&branch=pr-en-us-3828) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)
[dsc/resources/authoringResourceMofCS.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/dsc/resources/authoringResourceMofCS.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/resources/authoringresourcemofcs?branch=pr-en-us-3828) |
[wmf/5.0/limitation_dsc.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.0/limitation_dsc.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.0/limitation_dsc?branch=pr-en-us-3828) |
[wmf/5.0/odata_overview.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.0/odata_overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.0/odata_overview?branch=pr-en-us-3828) |
[wmf/5.1/Index.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/Index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/index?branch=pr-en-us-3828) |
[wmf/5.1/bugfixes.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/bugfixes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/bugfixes?branch=pr-en-us-3828) |
[wmf/5.1/community-resources.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/community-resources.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/community-resources?branch=pr-en-us-3828) |
[wmf/5.1/compatibility.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/compatibility.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/compatibility?branch=pr-en-us-3828) |
[wmf/5.1/install-configure.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/install-configure.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/install-configure?branch=pr-en-us-3828) |
[wmf/5.1/known-issues-cache/template.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/known-issues-cache/template.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/known-issues-cache/template?branch=pr-en-us-3828) |
[wmf/5.1/package-management-improvements.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/package-management-improvements.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/package-management-improvements?branch=pr-en-us-3828) |
[wmf/5.1/release-notes.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/release-notes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/release-notes?branch=pr-en-us-3828) |

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_windows_powershell_5.1.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C10%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903102019576754-3828%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yO61DuGLJYDmFscRUDgly9FuhVQB7jvgC%2F4BtHMNdVI%3D&st=2019-03-10T20%3A29%3A37Z&se=2019-04-10T20%3A34%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md (hunk 1 above)
Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: Weird. Thanks for the fix.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c185ece](https://github.com/nschonni/PowerShell-Docs/pull/3828/commits/c185ecea2a7ddb90889950d99f8c44c31e5185d9)_: 

:clock10: **Pending**: waiting for processors ([27](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **OPS Build is busy**, currently there are 27 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c185ece](https://github.com/nschonni/PowerShell-Docs/pull/3828/commits/c185ecea2a7ddb90889950d99f8c44c31e5185d9)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_powershell_exe?view=powershell-3.0&branch=pr-en-us-3828) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_windows_powershell_5.1?view=powershell-5.1&branch=pr-en-us-3828) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_windows_powershell_5.1.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_remote_requirements?view=powershell-6&branch=pr-en-us-3828) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)
[dsc/resources/authoringResourceMofCS.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/dsc/resources/authoringResourceMofCS.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/resources/authoringresourcemofcs?branch=pr-en-us-3828) |
[wmf/5.0/limitation_dsc.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.0/limitation_dsc.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.0/limitation_dsc?branch=pr-en-us-3828) |
[wmf/5.0/odata_overview.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.0/odata_overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.0/odata_overview?branch=pr-en-us-3828) |
[wmf/5.1/Index.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/Index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/index?branch=pr-en-us-3828) |
[wmf/5.1/bugfixes.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/bugfixes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/bugfixes?branch=pr-en-us-3828) |
[wmf/5.1/community-resources.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/community-resources.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/community-resources?branch=pr-en-us-3828) |
[wmf/5.1/compatibility.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/compatibility.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/compatibility?branch=pr-en-us-3828) |
[wmf/5.1/install-configure.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/install-configure.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/install-configure?branch=pr-en-us-3828) |
[wmf/5.1/known-issues-cache/template.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/known-issues-cache/template.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/known-issues-cache/template?branch=pr-en-us-3828) |
[wmf/5.1/package-management-improvements.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/package-management-improvements.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/package-management-improvements?branch=pr-en-us-3828) |
[wmf/5.1/release-notes.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/wmf/5.1/release-notes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/release-notes?branch=pr-en-us-3828) |

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_exe.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_windows_powershell_5.1.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/5.1/Microsoft.PowerShell.Core/About/about_windows_powershell_5.1.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/nschonni/PowerShell-Docs/blob/fix--MD003/heading-style/header-style/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903111756555679-3828%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=J4sUKQidPH6OqN867To9RsdBJMMyPYyjGHWBfa8Fiew%3D&st=2019-03-11T18%3A07%3A34Z&se=2019-04-11T18%3A12%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**